dgraphtest/local_cluster.go

```diff
@@ -433,3 +437,44 @@ func (c *LocalCluster) AssignUids(num uint64) error {
 	}
 	return nil
 }
+
+// CompareCommits compare given commit with current cluster's commit and return
```

Comment:
This statement and the function name could be simpler. Not sure which commit is the "cluster's commit", the names have to be clear.

CompareCommits -> IsParent(ancestor, descendant string) (bool, error)


// IsParent checks whether `ancestor` is the ancestor commit of the given descendant commit.